lab_5_scrapper/scrapper.py

```diff
@@ -120,46 +254,96 @@ def __init__(self, full_url: str, article_id: int, config: Config) -> None:
         """
         Initializes an instance of the HTMLParser class
         """
-        pass
+        self.full_url = full_url
+        self.article_id = article_id
+        self.config = config
+        self.article = Article(url=full_url, article_id=article_id)
 
     def _fill_article_with_text(self, article_soup: BeautifulSoup) -> None:
         """
         Finds text of article
         """
-        pass
+        texts_bs = article_soup.find('div', {
+            'class':
+                'elementor-element elementor-element-6732342 elementor-widget elementor-widget-theme-post-content'})
+        texts = texts_bs.find_all('p')
+        result = ''
+        for el in texts:
+            txt = el.text
+            txt = txt.replace('\n\t\t\t\t', '')
+            txt = txt.replace('\t\t\t', '')
+            if txt != '\xa0':
+                result = ' '.join([result, txt])
+        self.article.text = result
 
     def _fill_article_with_meta_information(self, article_soup: BeautifulSoup) -> None:
         """
         Finds meta information of article
         """
-        pass
+        title_bs = article_soup.find('h1', {'class': 'elementor-heading-title elementor-size-default'})
```

Comment:
is there a better way to find date? sequence of classes is fragile. (here and further)

Reply:
done